app/views/pets/read.html.erb

```diff
@@ -10,6 +10,6 @@
 <%= @pet.adopt_status %> adoption.
 <%= button_to "Change to Adoptable", "/pets/#{@pet.id}/adoptable", method: :patch %>
 <% end %></h3>
-
+<%= button_to "Favorite Pet", "pets/#{@pet.id}/favorite", method: :get %> 
```

Comment:
Done with Favorites, well done on the placement of button looks great on Heroku.